docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,83 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+Boto3 supports using proxies as intermediaries between your code and AWS. Proxies may provide any number of functions, from filtering to security and firewalls to privacy assurance.
+
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used when communicating between specific endpoints, or when
+using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary that maps
+the protocol name or endpoint URL to the address of a proxy server.
```

Comment:
We should clarify this is intended to only be a scheme for the key. If the endpoints keys are working right now, that's unintentional and I'm not sure we want to document it as supported. Where were you seeing that setup being used?